msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -96,17 +96,13 @@ You can enable or disable [group owner consent](/azure/active-directory/manage-a
 
 In addition, you can enable or disable group owner consent using PowerShell, follow the steps outlined in [configure group owner consent using PowerShell](/azure/active-directory/manage-apps/configure-user-consent-groups?tabs=azure-powershell).
 
-### Configure user consent settings for RSC in a chat
+### Configure chat owner consent settings for RSC in a chat (via Microsoft Graph APIs)
 
-You can enable or disable [user consent](/azure/active-directory/manage-apps/configure-user-consent?tabs=azure-portal) directly within the Azure portal:
+You can enable or disable RSC for chats using Graph API. The property `isChatResourceSpecificConsentEnabled` in [**teamsAppSettings**](/graph/api/teamsappsettings-update#example-1-enable-installation-of-apps-that-require-resource-specific-consent-in-chats-meetings) governs whether chat RSC is enabled in the tenant.
 
-1. Sign in to the [Azure portal](https://portal.azure.com) as a [Global Administrator or Company Administrator](/azure/active-directory/roles/permissions-reference#global-administrator&preserve-view=true).
-1. Select **Azure Active Directory** > **Enterprise applications** > **Consent and permissions** > [**User consent settings**](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ConsentPoliciesMenuBlade/UserSettings).
-1. Enable, disable, or limit user consent with the control labeled **User consent for applications**. The default is **Allow user consent for apps**. For a chat member to install an app using RSC, user consent must be enabled for that user.
-
-    ![Azure RSC chat configuration](../../assets/images/azure-rsc-chat-configuration.png)
+   ![Graph RSC team configuration](../../assets/images/rsc/graph-rsc-chat-configuration.png)
 
-In addition, you can enable or disable user consent using PowerShell, follow the steps outlined in [configure user consent using PowerShell](/azure/active-directory/manage-apps/configure-user-consent?tabs=azure-powershell).
+>  The default value of the property **isChatResourceSpecificConsentEnabled** is based on whether [user consent settings](/azure/active-directory/manage-apps/configure-user-consent?tabs=azure-portal) is turned on/off in the tenant when RSC for chats is first used. This could be the first time retrieving [**teamsAppSettings**](/graph/api/teamsappsettings-get) or installing a Teams app with resource-specific permissions in a chat/meeting.
```

Comment:
The default value of the property **isChatResourceSpecificConsentEnabled** is based on whether [user consent settings](/azure/active-directory/manage-apps/configure-user-consent?tabs=azure-portal) is turned on or off in the tenant when RSC for chats is first used. This can be during the first time retrieving [**teamsAppSettings**](/graph/api/teamsappsettings-get) or installing a Teams app with resource-specific permissions in a chat/meeting.

Reply:
Taken both.

Reply:
Updated.